plugins/defaultplugins/ifplugin/ifaceidx/dhcp_index_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ifaceidx_test
+
+import (
+	"testing"
+
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/ligato/vpp-agent/idxvpp"
+	"github.com/ligato/vpp-agent/idxvpp/nametoidx"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/ifaceidx"
+	. "github.com/onsi/gomega"
+)
+
+const (
```

Comment:
please do not use global variables in unit tests (even unexported). It's not a good practice, we agreed to not going to use it and we want to keep similar look & feel across the unit tests. 
If you want, define variables inside the function (which will be scoped to the function only)